verilog/formatting/formatter.cc

```diff
@@ -413,6 +419,19 @@ Status Formatter::Format(const ExecutionControl& control) {
     }
   }
 
+  {
```

Comment:
This location is good, it is clear that this is a transformational pass separate from the phases around it.

Reply:
Okay. Thanks!